### PR TITLE
Moving S390X to SOC7

### PR DESCRIPTION
**Pull request by aplanas**

### CODE REVIEW

File: chef/data_bags/crowbar/template-nova.json [hunk 1]
```diff
@@ -86,14 +86,13 @@
...
-        "zvm_xcat_network": "admin",
+        "zvm_xcat_username": "admin",
+        "zvm_xcat_password": "admin",
         "zvm_diskpool": "",
         "zvm_diskpool_type": "",
         "zvm_host": "",
-        "zvm_scsi_pool": "",
-        "zvm_user_profile": "",
+        "zvm_scsi_pool": "xcatzfcp",
+        "zvm_user_profile": "OSDFLT",
```
Comment (nicolasbock) [inline chef/data_bags/crowbar/template-nova.json]: Where do those come from?

**Comment by dirkmueller** [comment]
@aplanas rebase?

**Comment by aplanas** [comment]
Requires https://github.com/crowbar/crowbar-core/pull/1351

**Comment by aplanas** [comment]
[P2: Adapt Cinder barclamp to detect zVM name schema](https://trello.com/c/ZSak1h6X/136-p2-adapt-cinder-barclamp-to-detect-zvm-name-schema)

**Comment by rsalevsky** [comment]
Do we still need this?

**Comment by aplanas** [comment]
yep

**Pull request closed without merging**